tests/unit/importer/test_workbook_importer.py

```diff
@@ -1,10 +1,10 @@
 from unittest import TestCase
 from unittest.mock import MagicMock, patch
 
-from ingest.importer.conversion.metadata_entity import MetadataEntity
-from ingest.importer.importer import WorksheetImporter, WorkbookImporter, UnexpectedEntityUUIDFound, \
+from hca_ingest.importer.conversion.metadata_entity import MetadataEntity
+from hca_ingest.importer.importer import WorksheetImporter, WorkbookImporter, UnexpectedEntityUUIDFound, \
     MissingEntityUUIDFound, MultipleProjectsFound, NoProjectFound
-from ingest.importer.spreadsheet.ingest_workbook import IngestWorkbook
+from hca_ingest.importer.spreadsheet.ingest_workbook import IngestWorkbook
 
 from tests.utils import create_test_workbook, create_ingest_workbook
 
```

Comment:
It is possible to apply the same patch to all test methods.
see https://docs.python.org/3/library/unittest.mock-examples.html#applying-the-same-patch-to-every-test-method

```suggestion
@patch('hca_ingest.importer.importer.WorksheetImporter')
class WorkbookImporterTest(TestCase):

```

Reply:
This seems like a good idea, but how do we reference the patched object?
Most of the test methods are taking the patched object to set / detect return values.